src/main/java/seedu/uninurse/model/person/Patient.java

```diff
@@ -30,6 +30,28 @@ public Patient(Name name, Phone phone, Email email, Address address, ConditionLi
         this.tasks = tasks;
     }
 
+    /**
+     * Used to return a new immutable {@code Patient} when {@code ConditionList} is updated.
+     * @param patient the patient to be updated
+     * @param condupdatedConditionsitions the updated conditions.
```

Comment:
There is a typo here
```suggestion
     * @param updatedConditions the updated conditions.
```

Reply:
Fixed.